File: software/examples/csharp/ExampleSimpleVoltage.cs
using System;
using Tinkerforge;

class Example
{
	private static string HOST = "localhost";
	private static int PORT = 4223;
	private static string UID = "XYZ"; // Change XYZ to the UID of your Industrial Analog Out Bricklet 2.0

	static void Main()
	{
		IPConnection ipcon = new IPConnection(); // Create IP connection
		BrickletIndustrialAnalogOutV2 iao =
		  new BrickletIndustrialAnalogOutV2(UID, ipcon); // Create device object

		ipcon.Connect(HOST, PORT); // Connect to brickd
		// Don't use device before ipcon is connected

		// Set output voltage to 3.3V
		iao.SetVoltage(3300);
		iao.SetEnabled(true);

		Console.WriteLine("Press enter to exit");
		Console.ReadLine();
		iao.SetEnabled(false);
		ipcon.Disconnect();
	}
}

File: software/examples/rust/example_simple_voltage.rs
use std::{io, error::Error};

use tinkerforge::{ip_connection::IpConnection, 
                  industrial_analog_out_v2_bricklet::*};


const HOST: &str = "localhost";
const PORT: u16 = 4223;
const UID: &str = "XYZ"; // Change XYZ to the UID of your Industrial Analog Out Bricklet 2.0.

fn main() -> Result<(), Box<dyn Error>> {
    let ipcon = IpConnection::new(); // Create IP connection.
    let iao = IndustrialAnalogOutV2Bricklet::new(UID, &ipcon); // Create device object.

    ipcon.connect((HOST, PORT)).recv()??; // Connect to brickd.
    // Don't use device before ipcon is connected.

		// Set output voltage to 3.3V
		iao.set_voltage(3300);
		iao.set_enabled(true);

    println!("Press enter to exit.");
    let mut _input = String::new();
    io::stdin().read_line(&mut _input)?;
		iao.set_enabled(false);
    ipcon.disconnect();
    Ok(())
}
